test/integration/model/create.test.js

```diff
@@ -131,7 +131,7 @@ describe(Support.getTestDialectTeaser('Model'), () => {
           }
         }).catch(error => {
           expect(error).to.be.instanceof(Sequelize.UniqueConstraintError);
-          if (dialect === 'mysql') {
+          if (_.includes(['mysql', 'postgres'], dialect)) {
```

Comment:
interchange if statements and use `dialect === sqilite`, this will fail for `postgres-native` and any new dialects

Reply:
I modified the assertion so as to check the conflicting field in the errors property (array) of the UniqueConstrainError object, since checking for the fields property would also cause the mssql tests to fail when run by AppVeyor (and in this case I couldn't get any details from the error).